lib/pos_random_seed.go

```diff
@@ -64,17 +64,27 @@ func (randomSeedHash *RandomSeedHash) isEmpty() bool {
 // UTXO VIEW UTILS
 //
 
-func (bav *UtxoView) GenerateRandomSeedSignature(signerPrivateKey *bls.PrivateKey) (*bls.Signature, error) {
+func (bav *UtxoView) GenerateCurrentRandomSeedSignature(signerPrivateKey *bls.PrivateKey) (*bls.Signature, error) {
```

Comment:
It's actually the next random seed signature, which is built iteratively from the current random seed hash
```suggestion
func (bav *UtxoView) GenerateNextRandomSeedSignature(signerPrivateKey *bls.PrivateKey) (*bls.Signature, error) {
```